packages/repmgr13/repmgr.spec

```diff
@@ -41,9 +45,19 @@ Requires(postun):   initscripts
 Group:        Applications/Databases
 BuildRoot:        %{_tmppath}/%{name}-%{version}%{extra_version}-%{release}-root-%(%{__id_u} -n)
 %endif
-BuildRequires:    postgresql, postgresql-devel, postgresql-static
-BuildRequires:    libxslt-devel, pam-devel, openssl-devel, readline-devel
-Requires:         postgresql-server
+BuildRequires:    postgresql, postgresql-devel
+BuildRequires:    libxslt-devel, pam-devel, openssl-devel, readline-devel flex
```

Comment:
```suggestion
BuildRequires:    libxslt-devel, pam-devel, openssl-devel, readline-devel, flex
```

Reply:
😆 🤷🏻‍♂️

Reply:
hah, it wasn't me

c13f08867a461b6cf6454644a24c99aff8ef443a

I'll fix it from upstream